### PR TITLE
Change hands configuration files

### PR DESCRIPTION
**Pull request by Nicogene**
It requires https://github.com/robotology/gazebo-yarp-plugins/pull/662

cc @xEnVrE

### CODE REVIEW

**Review by xEnVrE — CHANGES_REQUESTED** [review]
Thanks @Nicogene, I am requesting changes as some of the numbers are not correct as per the documentation.

Moreover, the limits for the new hand in https://github.com/icub-tech-iit/ergocub-software/blob/5ca85d3b065f47ea652bb413d9b0536f5dfcc72c/urdf/creo2urdf/data/ergocub1_1/ERGOCUB_joint_all_parameters.csv seems not updated as in the documentation https://icub-tech-iit.github.io/documentation/hands/hands_mk5/

File: urdf/ergoCub/conf/gazebo_ergocub_left_hand_mk5_1.ini [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+[COUPLING_PARAMS]
+L0x      -0.0050 -0.0050 -0.0050 -0.0050  -0.0050
+L0y       0.0040  0.0040  0.0040  0.0040   0.0040
+q2bias    173.35  173.35  173.35  173.35  -170.54
+q1off       2.45    2.15    2.15    2.15     2.46
+k         0.0171 0.02918 0.02918 0.02918  0.02425
+d        0.02006 0.03004 0.03004 0.03004  0.02504
+l         0.0085 0.00604 0.00604 0.00604  0.00608
+b        0.00652  0.0055  0.0055  0.0055   0.0055
```
[the remark this answers is not in the document]
Comment (xEnVrE) [reply urdf/ergoCub/conf/gazebo_ergocub_left_hand_mk5_1.ini:16]: See https://github.com/icub-tech-iit/documentation/issues/307

File: urdf/ergoCub/conf/gazebo_ergocub_right_hand_mk5_1.ini [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+[COUPLING_PARAMS]
+L0x      -0.0050 -0.0050 -0.0050 -0.0050  -0.0050
+L0y       0.0040  0.0040  0.0040  0.0040   0.0040
+q2bias    173.35  173.35  173.35  173.35  -170.54
+q1off       2.45    2.15    2.15    2.15     2.46
+k         0.0171 0.02918 0.02918 0.02918  0.02425
+d        0.02006 0.03004 0.03004 0.03004  0.02504
+l         0.0085 0.00604 0.00604 0.00604  0.00608
+b        0.00652  0.0055  0.0055  0.0055   0.0055
```
[the remark this answers is not in the document]
Comment (xEnVrE) [reply urdf/ergoCub/conf/gazebo_ergocub_right_hand_mk5_1.ini:16]: https://github.com/icub-tech-iit/documentation/issues/307

File: urdf/ergoCub/conf/gazebo_ergocub_right_hand_mk5_1.ini [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+yarpDeviceName right_hand_hardware_device
+jointNames r_thumb_add r_thumb_prox r_thumb_dist r_index_add r_index_prox r_index_dist r_middle_prox r_middle_dist r_ring_prox r_ring_dist r_pinkie_prox r_pinkie_dist
+
+[COUPLING]
+icub_hand_mk5 (0 1 2 3 4 5 6 7 8 9 10 11) (r_thumb_add r_thumb_oc r_index_add r_index_oc r_middle_oc r_ring_pinky_oc reserved reserved reserved reserved reserved reserved) (0.0 0.0 0.0 0.0 0.0 0.0) (100.0 90.0 15.0 90.0 90.0 90.0)
+
+[COUPLING_PARAMS]
+L0x      -0.0050 -0.0050 -0.0050 -0.0050  -0.0050
+L0y       0.0040  0.0040  0.0040  0.0040   0.0040
+q2bias    173.35  173.35  173.35  173.35  -170.54
```
[the remark this answers is not in the document]
Comment (xEnVrE) [reply urdf/ergoCub/conf/gazebo_ergocub_right_hand_mk5_1.ini:11]: As per https://icub-tech-iit.github.io/documentation/hands/hands_mk5_coupling/#mk51-coupling-variables

**Comment by Nicogene** [comment]
@xEnVrE I updated the limits for the hand mk5.1 also in the csv/urdf and I corrected the coupling params as done here:

https://github.com/icub-tech-iit/documentation/pull/312

I think we can safely merge

**Review by xEnVrE — APPROVED** [review]

**Pull request merged**